CONTRIBUTING.md

```diff
@@ -80,12 +80,38 @@ For `static files`, the name pattern is `<module_name>.ext` (i.e.
 ...). Don't link data (image, libraries) outside Odoo: don't use an url to an
 image but copy it in our codebase instead.
 
+### Installation hooks
+When **`pre_init_hook`**, **`post_init_hook`** and **`uninstall_hook`** are
+used, they should be placed in **`hooks.py`** located at the root of module
+directory structure and keys in the manifest file keeps the same as the
+following
+
+```python
+{
+    ...
+    'pre_init_hook': 'pre_init_hook',
+    'post_init_hook': 'post_init_hook',
+    'uninstall_hook': 'uninstall_hook',
+    ...
+}
+```
+
+Remember to add into the **`__init__.py`** the following imports as needed
```

Comment:
@osvalr 
Use `*` is a bad idea.
We should use the name of `pre_init_hook`, `post_init_hook` and `uninstall_hook`
Because `import *` should be extra effects and show lint errors